cmake/ExternalProjects.cmake

```diff
@@ -31,6 +31,8 @@ endif()
 conan_cmake_configure(
     REQUIRES
         "catch2/2.13.7@#31c8cd08e3c957a9eac8cb1377cf5863"
+        "jwt-cpp/0.6.0@#cd6b5c1318b29f4becaf807b23f7bb44"
```

Comment:
The `jwt-cpp` dependency is needed to validate the JWT returned by the Azure Attestation Service. The `picojson` dependency _should_ be installed as a dependency of `jwt-cpp`, but there seems to be a problem in their conan recipe. I have opened a PR in the `jwt-cpp` repo.

Reply:
Looks like that's been merged, can we lose the `picojson` dep here now?

Reply:
Could you add a comment above here saying that these dependencies are only needed for the Azure attestation service?